exonum-java-binding/launcher-plugins/README.md

```diff
@@ -0,0 +1,96 @@
+# Java Runtime Plugins for Exonum Launcher
+
+## Installation
+
+#### Requirements
+
+- [Python3](https://www.python.org/downloads/)
+
+```bash
+# Generate sources
+mvn generate-sources -pl core
+# Install plugin
+python3 -m pip install -e launcher-plugins
+```
+
+## Usage
+
+TODO: Move the entire section to the EJB App tutorial.
```

Comment:
Is this going to be implemented within this PR?

Reply:
Probably not, the entire readme isn't even a part of this task.